inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ConsumptionController.java

```diff
@@ -64,9 +65,9 @@ public Response<ConsumptionSummary> getSummary(ConsumptionQuery query) {
 
     @GetMapping("/consumption/list")
     @ApiOperation(value = "List data consumptions")
-    public Response<PageInfo<ConsumptionListVo>> list(ConsumptionQuery query) {
+    public Response<PageResult<ConsumptionListVo>> list(ConsumptionQuery query) {
         query.setUsername(LoginUserUtils.getLoginUser().getName());
-        return Response.success(consumptionService.list(query));
+        return Response.success(PageUtils.of(consumptionService.list(query)));
```

Comment:
Why not encapsulate it directly into PageResult when querying in the Service layer?

There is one more conversion operation of PageUtils, which is not very efficient.